__tests__/app.test.js

```diff
@@ -68,19 +70,55 @@ describe("GET /api/articles/:article_id", () =>{
     .expect(404)
     .then(({body}) =>{
       const error = body
-      expect(error.msg).toBe("Couldn't find article")
+      expect(error.msg).toBe("Not found")
     })
   })
   })
 
+describe("GET /api/articles", ()=>{
+  test("200: Responds with an array of all the articles with all the correct properties, sorted by the date created in descending order", ()=>{
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(({body}) =>{
+      const articles = body.articles
+      expect(articles).toHaveLength(13)
+      articles.forEach((article)=>{
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(Number),
+
+        })
+      })
+      expect(articles).toBeSortedBy("created_at", { descending: true, })
+    })
+  })
+})
```

Comment:
Great test, I like the length check too. Since you've used ".toMatchObject" consider checking that body is _not_ there. 🙂